auditendpoints.py

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
I'm not sure if this file should have been added to the PR.

Reply:
Oops! I thought this was in my global git ignore but I guess it snuck through. Will remove